reconcile/utils/terraform_client.py

```diff
@@ -267,6 +267,7 @@ def log_plan_diff(
         always_enabled_deletions = {
             "random_id",
             "aws_lb_target_group_attachment",
+            "cloudflare_record",  # This is because a zone can contain up to one thousand records and it's not ot practical to require adding record to deletionApprovals
```

Comment:
```suggestion
            "cloudflare_record",  # This is because a zone can contain up to one thousand records and it's not practical to require adding each record to deletionApprovals
```